src/DynamoCore/Models/RecordableCommands.cs

```diff
@@ -504,6 +504,17 @@ internal static OpenFileCommand DeserializeCore(XmlElement element)
             protected override void ExecuteCore(DynamoModel dynamoModel)
             {
                 dynamoModel.OpenFileImpl(this);
+
+                // Log file open action and the number of nodes in the opened workspace
+                Dynamo.Logging.Analytics.CreateFileOperationEvent(
```

Comment:
Should we rename this method as TrackFileOperationEvent and still return IDisposable?